synapseclient/models/mixins/table_components.py

```diff
@@ -363,7 +363,7 @@ async def _table_query(
 
     - For `csv`, you can specify:
         - `quote_character`: Character used for quoting fields. Default is double quote (").
-        - `escape_character`: Character used for escaping special characters. Default is backslash (\).
+        - `escape_character`: Character used for escaping special characters. Default is backslash.
```

Comment:
I was seeing a warning about this since it's a special escape character, so I just removed it for now